lib/schema_doc/document.rb

```diff
@@ -6,7 +6,7 @@ class << self
       def read
         document = []
 
-        model_classes_group_by_table_name.sort.each do |table_name, model_classes|
+        model_classes_group_by_table_name.sort_by {|k, _| k }.each do |table_name, model_classes|
```

Comment:
I wonder if this change is required.  If `sort` was called without block, hash will maybe be sorted by key...?

```irb
[1] pry(main)> RUBY_VERSION
=> "2.3.1"
[2] pry(main)> {"bob" => [100], "charlie" => [200], "alice" => [500]}.sort
=> [["alice", [500]], ["bob", [100]], ["charlie", [200]]]
```